styles/main.css

```diff
@@ -0,0 +1,781 @@
+@font-face {
+  font-family: "COCOGOOSE";
+  font-style: normal;
+  font-weight: 400;
+  src: local("COCOGOOSE"), url("https://fonts.cdnfonts.com/s/26936/cocogoose_trial.woff") format("woff");
+}
+
+@font-face {
+  font-family: "Lato";
+  font-style: normal;
+  font-weight: 400;
+  src: url(https://fonts.gstatic.com/s/lato/v22/S6uyw4BMUTPHjx4wXg.woff2)format("woff2");
+}
+
+:root {
+  --main: #272a31;
+  --primary: #ee513b;
+  --primaryTransparent: #ee503b9c;
+  --blur: #d3d3d3;
+  --blur0: #f0f0f0;
+  --blur1: #919191;
+  --blur2: #686868;
+  --blur3: #3b3b3b;
+}
```

Comment:
The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design 👍

Reply:
@mateo951 Thank you. 😄